pyproject.toml

```diff
@@ -11,6 +11,7 @@ classifiers = ["License :: OSI Approved :: MIT License"]
 dynamic = ["version", "description"]
 dependencies = [
     "pyperclip==1.8.2",
+    "pytest-playwright>=0.3.3",
```

Comment:
is py still set to >=3.7? we might want to make this a more recent min requirement

it would also be nice to use uv from here on

Reply:
And use the new `uv publish`, let's try it :)